papas/detectors/CMS.py

```diff
@@ -9,9 +9,10 @@ class ECAL(DetectorElement):
     def __init__(self):
         volume = VolumeCylinder('ecal', 1.55, 2.1, 1.30, 2. )
         mat = material.Material('ECAL', 8.9e-3, 0.275)
-        self.eta_crack = 1.5
+        self.eta_crack = 1.479
         self.emin = {'barrel':0.3, 'endcap':1.}
-        self.eres = {'barrel':[0.073, 0.1, 0.005], 'endcap':[0.213, 0.224, 0.005]}
+        self.eres = {'barrel':[4.22163e-02, 1.55903e-01, 7.14166e-03], 'endcap':[-2.08048e-01, 3.25097e-01, 7.34244e-03]}
```

Comment:
@GaelTouquet what are these numbers in Luca's PR? 
They are not yet in my master, and I thought I had merged your work.